eng/CodeAnalysis.ruleset

```diff
@@ -1,8 +1,10 @@
 ﻿<RuleSet Name="Microsoft.Analyzers.ManagedCodeAnalysis" Description="Microsoft.Analyzers.ManagedCodeAnalysis" ToolsVersion="14.0">
-  <Rules AnalyzerId="Microsoft.Analyzers.ManagedCodeAnalysis" RuleNamespace="Microsoft.Rules.Managed">
+   <Rules AnalyzerId="Microsoft.CodeAnalysis.NetAnalyzers" RuleNamespace="Microsoft.CodeAnalysis.NetAnalyzers">
       <Rule Id="CA1000" Action="None" />             <!-- Do not declare static members on generic types -->
```

Comment:
You can get rid of majority of these `Action="None"` entries if you want - they should be disabled by default in the new package.

Reply:
Thanks.  I actually want to keep them for now so that this file includes all of the analyzers and makes it clear about which ones are enabled and which ones are disabled.